icefall/utils.py

```diff
@@ -199,26 +198,25 @@ def get_texts(best_paths: k2.Fsa) -> List[List[int]]:
       Returns a list of lists of int, containing the label sequences we
       decoded.
     """
-    if isinstance(best_paths.aux_labels, k2.RaggedInt):
+    if isinstance(best_paths.aux_labels, k2.RaggedTensor):
```

Comment:
@glynpu

Could you help to update k2 in ESPnet?

https://github.com/espnet/espnet/blob/26902eed11eb0ce143b99d2dbce922f3d1064e35/espnet2/bin/k2_asr_inference.py#L70

needs to be changed since there is no  `k2.RaggedInt` and `k2.RaggedFloat`
in k2 v1.7.

Reply:
Also, we have to install k2 v1.7 in ESPnet GitHub actions.